Assets/Mirror/Runtime/LocalClient.cs

```diff
@@ -91,9 +92,9 @@ void ProcessInternalMessages()
         internal void InvokeBytesOnClient(byte[] buffer)
         {
             // unpack message and post to internal list for processing
-            if (Protocol.UnpackMessage(buffer, out ushort msgType, out byte[] content))
+            if (Protocol.UnpackMessage(buffer, out ushort msgType, out NetworkReader contentReader))
```

Comment:
I would prefer to create the network reader here,  and pass it to unpackmessage.  for example:

```cs
NetworkReader reader = new NetworkReader(buffer);
if (Prototocol.UnpackMessage(reader, out short msgType)) 
{
    ...
   // continue using the reader
}
```